sdk/storage/Azure.Storage.Files/README.md

```diff
@@ -9,79 +9,178 @@ Windows, Linux, and macOS. Additionally, Azure file shares can be cached on
 Windows Servers with Azure File Sync for fast access near where the data is
 being used.
 
-[Source code][source] | [Package (NuGet)][package] | [API reference documentation][rest_docs] | [Product documentation][product_docs]
+[Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
 
 ## Getting started
+
 ### Install the package
+
 Install the Azure Storage Files client library for .NET with [NuGet][nuget]:
 
 ```Powershell
-Install-Package Azure.Storage.Files
+dotnet add package Azure.Storage.Files --version 12.0.0-preview.1
 ```
 
-**Prerequisites**: You must have an [Azure subscription][azure_sub], and a
+### Prerequisites
+
+You need an [Azure subscription][azure_sub] and a
 [Storage Account][storage_account_docs] to use this package.
 
-To create a Storage Account, you can use the [Azure Portal][storage_account_create_portal],
-[Azure PowerShell][storage_account_create_ps] or [Azure CLI][storage_account_create_cli]:
+To create a new Storage Account, you can use the [Azure Portal][storage_account_create_portal],
+[Azure PowerShell][storage_account_create_ps], or the [Azure CLI][storage_account_create_cli].
+Here's an example using the Azure CLI:
+
+```Powershell
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+```
 
 ## Key concepts
+
 Azure file shares can be used to:
+
 - Completely replace or supplement traditional on-premises file servers or NAS devices.
 - "Lift and shift" applications to the cloud that expect a file share to store file application or user data.
 - Simplify new cloud development projects with shared application settings, diagnostic shares, and Dev/Test/Debug tool file shares.
 
 ## Examples
-### Create a file share

```

Comment:
In other readme files you were adding the `using` statements too. At least for one example. I found it useful.
i.e. https://github.com/Azure/azure-sdk-for-net/tree/a8650518034730d88beb09956d428a9e2aa08ac9/sdk/storage/Azure.Storage.Blobs#uploading-a-blob

Reply:
Fixed